src/Microsoft.TestPlatform.ObjectModel/Constants.cs

```diff
@@ -173,13 +173,18 @@ public static class Constants
     /// <summary>
     /// The default protocol version
     /// </summary>
-    public static readonly ProtocolConfig DefaultProtocolConfig = new() { Version = 5 };
+    public static readonly ProtocolConfig DefaultProtocolConfig = new() { Version = 6 };
 
     /// <summary>
     /// The minimum protocol version that has debug support
     /// </summary>
     public const int MinimumProtocolVersionWithDebugSupport = 3;
 
+    /// <summary>
+    /// The minimum protocol version that has debug support
+    /// </summary>
+    public const int MinimumProtocolVersionWithCancelDiscoveryEventHandlerSupport = 6;
```

Comment:
We are not really consistent in using `static readonly` vs `const` in the codebase.